CMakeLists.txt

```diff
@@ -328,6 +328,7 @@ if(BUILD_SHARED_LIBS)
 
     file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/libwavpack.def ${FILE_CONTENTS})
     target_sources(wavpack PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/libwavpack.def)
+    target_link_libraries(wavpack PRIVATE -static-libgcc)
```

Comment:
```suggestion
    if(MINGW)
        target_link_libraries(wavpack PRIVATE -static-libgcc)
    endif()
```

Reply:
Also exclude MinGW toolchains using clang, like:
`if(MINGW AND NOT CMAKE_C_COMPILER_ID MATCHES "Clang"))`

Reply:
Thanks! I have incorporated these, except for the trailing paren in your suggestion @sezero ...

Reply:
>  except for the trailing paren in your suggestion 

Oops. sorry